core/services/chainlink/config.go

```diff
@@ -42,6 +42,33 @@ type Config struct {
 	Solana solcfg.TOMLConfigs `toml:",omitempty"`
 
 	Starknet stkcfg.TOMLConfigs `toml:",omitempty"`
+
+	Aptos []*RawConfig `toml:",omitempty"`
```

Comment:
This should use the ` RawConfigs` type from below but I'm not sure if the `RawConfig` type should be as a pointer or not